s3_management/manage.py

```diff
@@ -61,6 +62,7 @@ def nightly_packages_to_show(self: S3IndexType) -> Set[str]:
         # sorting, sorts in reverse to put the most recent versions first
         all_sorted_packages = sorted(
             {self.normalize_package_version(obj) for obj in self.objects},
+            key=lambda v: parse(v.split('-')[1]),
```

Comment:
Can we do something like that to make it more resilient to errors. 
```suggestion
            key=lambda v: parse(v.split('-', 1)[-1]),
```

Reply:
They should have the same effect because we only use the first two parts of split here: https://github.com/pytorch/builder/blob/master/s3_management/manage.py#L85